src/js/components/PodDetail.js

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
+import React from 'react';
+
+import Breadcrumbs from './Breadcrumbs';
+import Pod from '../structs/Pod';
+import PodHeader from './PodHeader';
+import TabsMixin from '../mixins/TabsMixin';
+
+const METHODS_TO_BIND = [
+];
+
+class PodDetail extends mixin(TabsMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.tabs_tabs = {
+      instances: 'Instances',
+      configuration: 'Configuration',
+      debug: 'Debug'
+    };
+
+    this.state = {
+      currentTab: Object.keys(this.tabs_tabs).shift()
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  renderConfigurationTabView() {
+    let {pod} = this.props;
+    return (
+      <pre>
+      &lt;PodConfigurationView pod={pod.getId()} /&gt;
+      </pre>
+    );
+  }
+
+  renderDebugTabView() {
+    let {pod} = this.props;
+    return (
+      <pre>
+      &lt;PodDebugTabView pod={pod.getId()} /&gt;
+      </pre>
+    );
+  }
+
+  renderInstancesTabView() {
+    let {pod} = this.props;
+    return (
+      <pre>
+      &lt;PodInstancesView pod={pod.getId()} /&gt;
+      </pre>
+    );
+  }
+
+  render() {
+    const {pod} = this.props;
+
+    return (
+      <div className="flex-container-col">
+        <div className={classNames(
```

Comment:
Let's just pass a string with all of these classnames since we're not conditionally applying any of them.